deployments/common/image/common-scripts/set-notebook-kernel

```diff
@@ -22,28 +22,35 @@ def set_kernel(kernel, display_name, notebook):
     with open(notebook, "w+") as nbf:
         nbf.write(json.dumps(loaded, indent=1))
 
+
 def parse_args():
-    parser = argparse.ArgumentParser(description='Set the default kernel and kernel display name in the specified Jupyter notebooks.')
-    parser.add_argument('kernel', type=str,
-                        help='Jupyter kernel to use for notebooks by default.')
-    parser.add_argument('notebooks', type=str, nargs='+',
-                        help='Notebooks to update')
+    parser = argparse.ArgumentParser(
+        description="Set the default kernel and kernel display name in the specified Jupyter notebooks."
+    )
+    parser.add_argument(
+        "kernel", type=str, help="Jupyter kernel to use for notebooks by default."
+    )
+    parser.add_argument("notebooks", type=str, nargs="+", help="Notebooks to update")
     args = parser.parse_args()
     return args
 
+
 def main():
     args = parse_args()
     with open(f"/opt/environments/{args.kernel}/kernel.name") as f:
         display_name = f.read().strip()
     for notebook in args.notebooks:
         if not notebook.startswith("#") and notebook.endswith(".ipynb"):
-            print(f"Setting kernel={args.kernel}  display_name={display_name}  notebook={notebook}")
+            print(
+                f"Setting kernel={args.kernel}  display_name={display_name}  notebook={notebook}"
+            )
```

Comment:
Is this type of nesting enforced by flake8?

Reply:
No.   That nesting was generated by the "black" source code formatter I ran on the script to take care of other whitespace issues flake8 did care about.